app/Livewire/Project/Service/EditDomain.php

```
@@ -18,14 +18,25 @@ class EditDomain extends Component
 
+    public ?string $fqdn = null;
+
     protected $rules = [
-        'application.fqdn' => 'nullable',
-        'application.required_fqdn' => 'required|boolean',
+        'fqdn' => 'nullable',
     ];
 
     public function mount()
     {
         $this->application = ServiceApplication::find($this->applicationId);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\ServiceApplication' in method 'mount'. (undefined)

(StaticAccess)